src/vampyr/treebuilders/project.h

```diff
@@ -18,6 +19,15 @@ template <int D> void project(pybind11::module &m) {
         .def(
             "__call__",
             [](PyScalingProjector<D> &P, std::function<double(const Coord<D> &r)> func) {
+
+                try {
+                    auto arr = std::array<double, D>();
+                    arr.fill(111111.111); // A number which hopefully does not divide by zero
+                    func(arr);
+                } catch (py::cast_error &e) {
+                    py::print("Error: Invalid definition of analytic function");
+                    throw;
+                }
```

Comment:
what's this block supposed to do?

Reply:
It catches a `cast_error` which kills `Jupyter kernels`.  Want a comment there?

Reply:
Added a comment